packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -59,6 +68,21 @@ export class VaAdditionalInfo {
     }
   }
 
+  // Ensures that the CSS animation is consistent and uses the correct max-height for its content
+  updateInfoMaxHeight() {
+    const infoElm = this.el.shadowRoot.getElementById('info');
+    const contentHeight = infoElm.scrollHeight + 'px';
+    // the additional 2em is #info margin-top and margin-bottom when open
+    infoElm.style.setProperty(
+      '--calc-max-height',
+      'calc(' + contentHeight + ' + 2rem)',
+    );
+  }
```

Comment:
`scrollHeight` includes the element's padding, but not its border, margin or horizontal scrollbar ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight)).